interview-questions.md

```diff
@@ -15,54 +15,54 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is a code management software. Git is software and Github is the technology that runs git. 
 
-  Researched answer:
+  Researched answer: Git is a software that is in charge of monitoring the changes in software code, whose original code is available to be modified by other users. The main difference between the two is that Github is closed source, while Git is open source. The other difference is that Git is software, whereas Github is a cloud service that hosts Git.  
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: <, >, <=, >=, ==, ===, !=, ||, &&
 
-  Researched answer:
+  Researched answer: Logical operators test for true or false and return a Boolean value. 
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: An index is a specific location of an element within an object. A value specifies an input and would be the equivalent to said object. 
 
-  Researched answer:
+  Researched answer: Index represents the location of an element within a value. An example would be the letter "d" in the word "index". It is the third letter in the word. As such, its index would be 3, for its location within the word. The word "index" would be considered a value. The letters of the word would be considered elements. The location of those letters, within the word, would be considered indexes.  
```

Comment:
Love the example!  This is what I would like to see if possible in your initial answer, as that will show an interviewer a deeper level of understanding.